src/Entity/Pia/ProcessedDataType.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * Copyright (C) 2015-2018 Libre Informatique
+ *
+ * This file is licensed under the GNU LGPL v3.
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace PiaApi\Entity\Pia;
+
+use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation as JMS;
+use PiaApi\Entity\Pia\Traits\ResourceTrait;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="processing__processing_data_type")
+ */
+class ProcessedDataType
+{
+    use ResourceTrait;
+
+    /**
+     * @ORM\Column(type="string")
+     * @JMS\Groups({"Default", "Export"})
+     *
+     * @var string
+     */
+    protected $specificDataRetentionPeriod;
```

Comment:
retentionPeriod?

Reply:
https://translate.google.com/?oe=utf-8&client=firefox-b-ab&um=1&ie=UTF-8&hl=fr&client=tw-ob#en/fr/data%20retention%20period :wink:
The translation may be wrong...